charts/hivemq-edge/values.schema.json

```
@@ -276,6 +280,44 @@
         ]
       }
     },
+    "modules": {
+      "dataHub": {
+        "description" : "Configuration for DataHub.",
+        "type" : "object",
+        "properties" : {
+          "enabled" : {
+            "type" : "boolean",
+            "description" : "Enable/Disable datahub."
+          },
+          "init" : {
+            "description" : "Specify the init file to load as for datahub.",
+            "type" : "string"
+          }
+        }
+      },
+      "persistence" : {
+        "description" : "Location of the edge persistence.",
+        "mode" : {
+          "enum" : [
+            "in-memory",
+            "file",
+            "file-native"
+          ],
+          "type" : "string"
+        },
+        "enabled" : {
+          "type" : "boolean"
+        },
+        "storageClassName" : {
+          "description" : "Storage class to be used.",
+          "type" : "object"
```

This is a string schema type
```suggestion
          "type" : "string"
```

---

fixed.